NEWS.md

```diff
@@ -1,5 +1,19 @@
 MXNet Change Log
 ================
+## 1.2.1
+### Deprecations
+- An incorrect [usage](https://github.com/apache/incubator-mxnet/issues/11091) of `save_params` was advertised in the gluon book which led to MXNet users depending on the incorrect usage and developing a hack around it. A change to internal structure of params file saved by `save_params` to resolve a bug, led to user scripts with the above mentioned hack to break. To fix this, we have reverted `save_params` and `load_params` to previous format and marked it as deprecated. We have added new APIs `save_parameters` and `load_parameters` for the new format. All scripts to save and load parameters for a Gluon model should now use the new API for `save_parameters` and `load_parameters`. If your model is hybridizable and you want to export a serialized structure of the model as well as parameters you need to use the `export` API and the newly added `imports` API instead of `save_params` and `load_params` API. For more details, Please see: [issue](https://github.com/apache/incubator-mxnet/issues/11091), [PR](https://github.com/apache/incubator-mxnet/pull/11127).
```

Comment:
->  A change was made to the internal structure of the `.params` file saved by `save_params` to resolve a bug. This led to user scripts with the above mentioned hack to break